### PR TITLE
[DOC-7623] Cloud Organization Audit Logs GA

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-7623] Cloud Organization Audit Logs GA

## Previews
[releases/cloud.md](https://deploy-preview-16932--cockroachdb-docs.netlify.app/docs/releases/cloud.html)
[cockroachcloud/cloud-org-audit-logs.md](https://deploy-preview-16932--cockroachdb-docs.netlify.app/docs/cockroachcloud/cloud-org-audit-logs.html)
[cockroachcloud/cluster-management.md](https://deploy-preview-16932--cockroachdb-docs.netlify.app/docs/cockroachcloud/cluster-management.html)
[cockroachcloud/cluster-overview-page.md](https://deploy-preview-16932--cockroachdb-docs.netlify.app/docs/cockroachcloud/cluster-overview-page.html)
[cockroachcloud/production-checklist.md](https://deploy-preview-16932--cockroachdb-docs.netlify.app/docs/cockroachcloud/production-checklist.html)
[v22.2/cockroachdb-feature-availability.md](https://deploy-preview-16932--cockroachdb-docs.netlify.app/docs/v22.2/cockroachdb-feature-availability.html)
[v23.1/cockroachdb-feature-availability.md](https://deploy-preview-16932--cockroachdb-docs.netlify.app/docs/v23.1/cockroachdb-feature-availability.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

**Review by abhinavg6 — APPROVED** [review]
LGTM

cc @rgcase @jramos-crl - This is the public doc to make Org Audit Logs GA next week. It will be published then only.

File: cockroachcloud/cluster-management.md [hunk 1]
```diff
@@ -191,7 +191,7 @@ All databases are not backed up at the same time. Each database is backed up eve
 
 {{ site.data.products.dedicated }} advanced clusters have a **PCI ready** panel to monitor the status of security features required for [PCI readiness](pci-dss.html). Feature statuses will update from **INACTIVE** to **ACTIVE** once you configure them. Learn more about configuring these features:
 
-- [Audit logs](cloud-org-audit-logs.html)
+- [{{ site.data.products.db }} Organization Audit logs](cloud-org-audit-logs.html)
```
Comment (taroface) [inline cockroachcloud/cluster-management.md:194]: These bullets should have a period per our style.

File: cockroachcloud/cluster-overview-page.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ The **Cluster configuration** panel displays the settings you chose during [clus
 
 {{ site.data.products.dedicated }} advanced clusters have a **PCI ready** panel to monitor the status of security features required for [PCI readiness](pci-dss.html). Feature statuses will update from **INACTIVE** to **ACTIVE** once you configure them. Learn more about configuring these features:
 
-- [Audit logs](cloud-org-audit-logs.html)
+- [{{ site.data.products.db }} Organization Audit logs](cloud-org-audit-logs.html)
```
Comment (taroface) [inline cockroachcloud/cluster-overview-page.md:86]: Ditto

File: cockroachcloud/production-checklist.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ To verify that you have authorized the application server's network, navigate to
 
 {{ site.data.products.dedicated }} advanced has access to all features required for [PCI readiness](pci-dss.html). You should configure these settings to make your cluster PCI ready:
 
-- [Audit logs](cloud-org-audit-logs.html)
+- [{{ site.data.products.db }} Organization Audit logs](cloud-org-audit-logs.html)
```
Comment (taroface) [inline cockroachcloud/production-checklist.md:39]: Ditto

**Review by taroface — APPROVED** [review]
LGTM, tiny comment on style

File: cockroachcloud/cluster-management.md (hunk 1 above)
Comment (mdlinville) [reply cockroachcloud/cluster-management.md:194]: I disagree in this case. There is not a single hard-and-fast rule about bullets and title vs sentence case; [it depends](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#lists). These are not sentences or sentence-like.

Comment (taroface) [reply cockroachcloud/cluster-management.md:194]: My mistake, you're right. I totally forgot that our style book leaves this open.

**Review by jramos-crl — APPROVED** [review]
LGTM

**Pull request merged**